src/App.tsx

```diff
@@ -29,8 +31,8 @@ export default class App extends React.Component <IProps, IState> {
 
         const playerStats = new PlayerStats();
         const eventTracker = new EventTracker(
-            [new BoomerGregorEvent()],
-            [new PickFacultyEvent()]
+            [new BoomerGregorEvent(), new CuteGirlEvent(), new FratPartyEvent()],
+            [new WelcomeEvent()]
```

Comment:
Seeing as though Eloise has a more involved starting event in her PR #80, it might be best to roll this back :(